test/test-utils.sh

```diff
@@ -31,7 +31,7 @@ TEST_DIR=testdir
 ALT_TEST_TEXT_FILE=test-s3fs-ALT.txt
 TEST_TEXT_FILE_LENGTH=15
 BIG_FILE=big-file-s3fs.txt
-TEMP_DIR="/var/tmp"
+TEMP_DIR=${TMPDIR:-"/var/tmp"}
```

Comment:
Would it be more clear to use the same variable:

```
TEMP_DIR="${TEMP_DIR:-/var/tmp}"
```

Reply:
Of course I can change this.

I decided to use TMPDIR because this variable is well-known and specified e.g. in the Single UNIX Specification. As admin I would try TMPDIR first.


Change it or leave it?